server/channels/api4/preference_test.go

```diff
@@ -623,6 +662,28 @@ func TestDeletePreferences(t *testing.T) {
 	resp, err = client.DeletePreferences(context.Background(), th.BasicUser.Id, preferences)
 	require.Error(t, err)
 	CheckUnauthorizedStatus(t, resp)
+
+	//DeletePreferences API from System Admin and Local Client
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, c *model.Client4) {
+
+		//Creating Test Data
+		var preferences model.Preferences
```

Comment:
CI doesn't like the style
```suggestion
	th.TestForSystemAdminAndLocal(t, func(t *testing.T, c *model.Client4) {
		//Creating Test Data
		var preferences model.Preferences
```

Reply:
Fixed the CI Issue